zulip_bots/zulip_bots/lib.py

```diff
@@ -198,6 +251,18 @@ def handle_message(message):
                 return
 
         if is_private_message or is_mentioned:
+            # Handle any default_commands first
+            if len(default_commands) > 0:
+                if '' in default_commands and len(message['content']) == 0:
+                    restricted_client.send_reply(message, default_commands[''][0]())
+                    return
+                for command in default_commands:
+                    if command == '':
+                        continue
+                    if message['content'].startswith(command):
```

Comment:
Maybe we want a simple equality check here, so we can simplify the whole `# Handle any default_commands first` if clause?